src/BugsnagUnity/Clients/AndroidClient.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Collections.Generic;
+using BugsnagUnity.Payload;
+using UnityEngine;
+
+namespace BugsnagUnity
+{
+  class AndroidClient : INativeClient
+  {
+    public IConfiguration Configuration { get; }
+
+    public IBreadcrumbs Breadcrumbs { get; }
+
+    public IDelivery Delivery { get; }
+
+    internal AndroidJavaObject JavaClient { get; }
+
+    public AndroidClient(AndroidConfiguration configuration)
+    {
+      Configuration = configuration;
+
+      using (var notifier = new AndroidJavaClass("com.bugsnag.android.Notifier"))
+      using (var info = notifier.CallStatic<AndroidJavaObject>("getInstance"))
+      {
+        info.Call("setURL", NotifierInfo.NotifierUrl);
+        info.Call("setName", NotifierInfo.NotifierName);
+        info.Call("setVersion", NotifierInfo.NotifierVersion);
+      }
+
+      using (var unityPlayerClass = new AndroidJavaClass("com.unity3d.player.UnityPlayer"))
+      using (var activity = unityPlayerClass.GetStatic<AndroidJavaObject>("currentActivity"))
+      using (var context = activity.Call<AndroidJavaObject>("getApplicationContext"))
+      {
+        JavaClient = new AndroidJavaObject("com.bugsnag.android.Client", context, configuration.JavaObject);
+
+        // the bugsnag-android notifier uses Activity lifecycle tracking to
+        // determine if the application is in the foreground. As the unity
+        // activity has already started at this point we need to tell the
+        // notifier about the activity and the fact that it has started.
+        using (var sessionTracker = JavaClient.Get<AndroidJavaObject>("sessionTracker"))
+        using (var activityClass = activity.Call<AndroidJavaObject>("getClass"))
+        {
+          var activityName = activityClass.Call<string>("getSimpleName");
+          sessionTracker.Call("updateForegroundTracker", activityName, true, 0L);
```

Comment:
This needs to be set to the Java time in ms, otherwise the created session won't be used in the Releases Dashboard.

Reply:
Im not sure what you mean by the Java time in ms. The branch of code this executes only does this `foregroundActivities.add(activityName);` so that parameter is unused.

Reply:
Passing in `0L` will result in no session being started within [updateForegroundTracker](https://github.com/bugsnag/bugsnag-android/blob/master/sdk/src/main/java/com/bugsnag/android/SessionTracker.java#L279). If that's intentional and a session is started via some other method then this is fine, otherwise we would need to pass in the current Date in milliseconds since the Java epoch.

Reply:
I only want this line of code to run `foregroundActivities.add(activityName);` session tracking is disabled in the android notifier as we do it in the c# code